netbox/ipam/models/vlans.py

```diff
@@ -273,3 +275,52 @@ def get_vminterfaces(self):
     @property
     def l2vpn_termination(self):
         return self.l2vpn_terminations.first()
+
+
+class VLANTranslationPolicy(PrimaryModel):
+    name = models.CharField(
+        verbose_name=_('name'),
+        max_length=100,
+        unique=True,
+    )
+    description = models.CharField(
+        verbose_name=_('description'),
+        max_length=200,
+        blank=True,
+    )
+
+    class Meta:
+        verbose_name = _('VLAN translation policy')
+        verbose_name_plural = _('VLAN translation policies')
+        ordering = ('name',)
+
+    def __str__(self):
+        return self.name
+
+
+class VLANTranslationRule(NetBoxModel):
+    policy = models.ForeignKey(
+        to=VLANTranslationPolicy,
+        related_name='rules',
+        on_delete=models.CASCADE,
+    )
+    local_vid = models.IntegerField()
+    remote_vid = models.IntegerField()
+
+    class Meta:
+        verbose_name = _('VLAN translation rule')
+        ordering = ('policy', 'local_vid', 'remote_vid',)
```

Comment:
If `local_vid` is unique per policy we can omit `remove_vid` from this set.
```suggestion
        ordering = ('policy', 'local_vid',)
```